raiden-ts/src/transport/epics/messages.ts

```diff
@@ -59,37 +58,29 @@ export function matrixMessageSendEpic(
     filter(isActionOf(messageSend.request)),
     // this mergeMap is like withLatestFrom, but waits until matrix$ emits its only value
     mergeMap((action) => matrix$.pipe(map((matrix) => ({ action, matrix })))),
-    groupBy(({ action }) => action.meta.address),
     // merge all inner/grouped observables, so different user's "queues" can be parallel
```

Comment:
This removes per-user queuing. Should we create an issue to bring it back?

Reply:
I don't think so. Queueing may be useful, but here, it was just causing overhead. Since `waitMemberAndSend$` will wait for the perfect set of conditions to align before attempting sending, and reactively waiting is cheaper than the queue and creates an _implicit_ queue (for the messages waiting to be sent), we don't need the explicit queue here anymore.

Reply:
Each element next'ed on this observable will end up and stop at this `waitMemberAndSend$`. This function returns an observable, which is also immediately subscribed, and which will only emit once the _set of conditions_ met (e.g. partner is online and has joined a room with us). Then, in the order these observables were created, they're resolved by the event which fulfilled them (e.g. the membership event when partner joined the room), and therefore this makes an implicit queue, where the async callbacks are called in order, but they're resolved **parallelly**, but which isn't a problem since we want all messages sent anyway.